client/signup/steps/domains/index.jsx

```diff
@@ -432,6 +433,36 @@ export class RenderDomainsStep extends Component {
 		);
 
 		this.props.setDesignType( this.getDesignType() );
+
+		// For the `domain-for-gravatar` flow, add an extra `domain_for_gravatar` property to the domain registration product,
+		// pre-select the "domain" choice in the "site or domain" step and skip the others, going straight to checkout
+		if ( this.props.flowName === 'domain-for-gravatar' ) {
+			const domainForGravatarItem = domainRegistration( {
+				domain: suggestion.domain_name,
+				productSlug: suggestion.product_slug,
+				extra: {
+					domain_for_gravatar: true,
```

Comment:
what do you think about `is_gravatar_domain`? to follow the `is_*` for booleans.